msteams-platform/mstdd-landing.yml

```diff
@@ -61,6 +61,8 @@ landingContent:
   linkLists:
     - linkListType: whats-new
       links:
+      - text: Tailor your app's behavior in applicable Microsoft 365 hosts (Developer preview)
```

Comment:
@erikadoyle, just thinking out loud here. Does this shorter version lend itself to ambiguity? Should we stick with the original sentence that mentions runtime requirements?

Reply:
How about this:

Specify runtime requirements so your app only runs in applicable Microsoft 365 hosts (Developer preview)